tema2/src/main/java/ro/sci/tema2/SalesRep.java

```diff
@@ -0,0 +1,77 @@
+package ro.sci.tema2;
+
+import java.util.Scanner;
+
+/**
+ * Created by camarasanvlad on 7/22/17.
+ */
+
+public class SalesRep {
+
+    public static void main(String[] args) {
+
+        /**
+         This method creates the list of objects based on created objects of Sales Representative with the following
+         characteristics: name, number of sales, quota of sales and worth of sales which is the product
+         between number of sales and quota of sales.).
+         */
+        SalesRepresentative[] salesRepresentatives = inputData();
+        System.out.println("\nThis is the list of sales representatives:");
+
+        /**
+         This method prints the list of Sales Representative in the order as they were introduced from the keyboard
+         (Sales Representative - name, number of sales, quota of sales and worth of sales which is the product
+         between number of sales and quota of sales.
+         */
+        printCollection(salesRepresentatives);
+
+        /**
+         This method is sorting the list of Sales Representative in descending order based on the amount of revenue
+         generated via sales. After the sorting, the first Sales Representative should be the one with the most
+         revenue generated. The last Sales Representative in the list should be the one with the least revenue generated.
+         */
+        sort(salesRepresentatives);
+        System.out.println("\nThis is the list of sales representatives in descending order based on the amount " +
+                "of revenue generated via sales:");
+        printCollection(salesRepresentatives);
+    }
+
+    private static void sort(SalesRepresentative[] salesRepresentatives) {
+        for (int k = 1; k < salesRepresentatives.length; k++) {
```

Comment:
This is not bubble sort. 
Bubble sort has a stop condition.
Please investigate the basic Bubble Sort algorithm.